correios/client.py

```diff
@@ -380,3 +421,39 @@ def get_tracking_code_events(self, tracking_list):
         response = self.websro.buscaEventosLista(self.username, self.password, "L", "T", "101",
                                                  tuple(tracking_codes.keys()))
         return self.model_builder.load_tracking_events(tracking_codes, response)
+
+    def calculate_freights(self,
+                           posting_card: PostingCard,
+                           services: List[Union[Service, int]],
+                           from_zip: Union[ZipCode, int, str], to_zip: Union[ZipCode, int, str],
+                           package: Package,
+                           value: Union[Decimal, float] = 0.00,
+                           extra_services: Optional[Sequence[Union[ExtraService, int]]] = None):
+
+        administrative_code = posting_card.administrative_code
+        services = [Service.get(s) for s in services]
+        from_zip = ZipCode.create(from_zip)
+        to_zip = ZipCode.create(to_zip)
+
+        if extra_services is None:
+            extra_services = []
+        else:
+            extra_services = [ExtraService.get(es) for es in extra_services]
```

Comment:
eu preciso da list comprehension e do [], ou seja, preciso do if *e* do else... ou faria um if-expression ou teria que usar `and` e `or`:

```python
extra_services = [ES.get(es) for es in extra_services] if extra_services else []
# or...
extra_services = extra_services and [ES.get(es) for es in extra_services] or []
```

Mas acho a legibilidade em ambos os casos bem ruim.

(além disso deve ter um bug no mypy que alerta para uma iteração em um objeto None)